src/storage/gpu_device_storage.h

```diff
@@ -43,20 +43,24 @@ class GPUDeviceStorage {
  public:
   /*!
    * \brief Allocation.
-   * \param size Size to allocate.
-   * \return Pointer to the storage.
+   * \param handle Handle struct.
    */
-  inline static void* Alloc(Storage::Handle* handle);
+  inline static void Alloc(Storage::Handle* handle);
   /*!
    * \brief Deallocation.
-   * \param ptr Pointer to deallocate.
+   * \param handle Handle struct.
    */
   inline static void Free(Storage::Handle handle);
 };  // class GPUDeviceStorage
 
-inline void* GPUDeviceStorage::Alloc(Storage::Handle* handle) {
-  const size_t size = handle->size;
+inline void GPUDeviceStorage::Alloc(Storage::Handle* handle) {
   void* ret = nullptr;
```

Comment:
Can we remove ret? why do we need i if we are allocating dptr?

Reply:
We can.